chain/ethhashlookup/eth_transaction_hash_lookup.go

```diff
@@ -0,0 +1,166 @@
+package ethhashlookup
```

Comment:
We can probably just name this file `table.go`, no need to repeat the package name.